modules/Movecraft/src/main/java/net/countercraft/movecraft/repair/RepairManager.java

```diff
@@ -21,6 +25,48 @@ public void run() {
         }
     }
 
+    public int convertOldCraftRepairStates(){
+        //Check in the old RepairStates folder
+        File repairStateDir = new File(Movecraft.getInstance().getDataFolder(),"RepairStates");
+        if (!repairStateDir.exists()){
+            return 0;
+        }
+        int convertedRepairStates = 0;
+        File[] repairStates = repairStateDir.listFiles();
+        if (repairStateDir.exists() && (repairStates != null || repairStates.length > 0)){
+            for (File rs : repairStates){
+                if (!rs.getName().contains(".schematic")){
+                    continue;
+                }
+                String fileName = rs.getName();
+                fileName = fileName.replace(".schematic", "");
+                OfflinePlayer owner = null;
+
+                for (OfflinePlayer op : Bukkit.getOfflinePlayers()){
+                    if (fileName.startsWith(op.getName())){
+                        owner = op;
+                        fileName = fileName.replace(op.getName(),"");
+                        break;
```

Comment:
the names fred and fredric can both exist. You'll have to keep track of how many times this succeeds and log an error instead of breaking